src/wallet/rpcwallet.cpp

```diff
@@ -2444,6 +2449,14 @@ static UniValue getwalletinfo(const JSONRPCRequest& request)
         obj.pushKV("hdseedid", seed_id.GetHex());
     }
     obj.pushKV("private_keys_enabled", !pwallet->IsWalletFlagSet(WALLET_FLAG_DISABLE_PRIVATE_KEYS));
+    if (pwallet->IsScanning()) {
+        UniValue scanning(UniValue::VOBJ);
+        scanning.pushKV("duration", pwallet->ScanningDuration() / 1000);
+        scanning.pushKV("progress", pwallet->ScanningProgress());
+        obj.pushKV("scanning", scanning);
+    } else {
+        obj.pushKV("scanning", UniValue());
```

Comment:
Seems better to just omit it?

Reply:
Yeah I could do that.

Reply:
Then the result would look the same as in the previous version and you couldn't tell (without looking at the bitcoin core version) whether a rescan is happening or not.

Reply:
Yeap, that's true if the client is human.

Reply:
IIRC `null` is the same as omitted in JSON-RPC, so maybe in that case we should put `false` here?

Reply:
yeah, the value is omitted, but not the key.

Though, I am also fine with "false", maybe it is more clear even

Reply:
Changed `null` to `false`.